dictionaries/medicalterms/checksum.txt

```diff
@@ -1,3 +1,3 @@
-8438032358b0863d385a64143b75ece05cf261d7  medicalterms-en.txt.gz
+6053fc00ec7a37da3a5b73eb772a69fa32921c41  medicalterms-en.txt.gz
```

Comment:
@KieranRobson,

This checksum should not have changed, please make sure all the dependencies are up to date: `pnpm i` from the root.

Reply:
Is it possible your fork is behind the main fork? There was a change in the tools that prevented the `.gz` files from being different if they were built on Linux, MacOS,or Windows.

Reply:
I believe this was because I was behind on the main branch